.github/workflows/ci.yml

```diff
@@ -2,10 +2,40 @@ name: 'CI Pipeline'
 on:
   push:
 jobs:
+  build:
+    needs: [install]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [20.10.0, 18.x]
```

Comment:
svelte-kit seems to have issues on some node versions, this ones working (20.10.0) so for now until we can update/fix we should just stick here